packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -556,6 +556,10 @@ export const handlers = [
       restricted: false,
       // Parent/Child: switch the `user_type` depending on what account view you need to mock.
       user_type: 'parent',
+      // PLACEMENT GROUPS TESTING - Permissions and Grants:
+      // Uncommnet the two lines below: This is important! The grants endpoint is only called for restricted users.
```

Comment:
```suggestion
      // Uncomment the two lines below: This is important! The grants endpoint is only called for restricted users.
```

(+ if you want to keep these commented changes for easier testing with mocks in the future, we'll need to resolve the merge conflicts stemming from the [MSW update PR](https://github.com/linode/manager/pull/10285))

Reply:
Thanks for catching this typo. The merge conflicts have been resolved.